### ISSUE TITLE
Make is possible to update QF Round banner in admin bro

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3665]
Every round, I need to create an issue to bug a frontend dev to update the banner... it would be better if I Could just set this myself in admin bro.

It should have multiple fields to adapt for responsiveness.

I think we need feedback from both @MohammadPCh & @mohammadranjbarz on how to do it.

cc @jainkrati

**Comment by mohammadranjbarz** [comment]
> Every round, I need to create an issue to bug a frontend dev to update the banner... it would be better if I Could just set this myself in admin bro.
> 
> It should have multiple fields to adapt for responsiveness.
> 
> I think we need feedback from both @MohammadPCh & @mohammadranjbarz on how to do it.
> 
> cc @jainkrati

From backend side it would take less than 2 hours, I can add a field named `banner` in qfRound, @MohammadPCh  is it ok?

**Comment by laurenluz** [comment]
tagging @CarlosQ96 in here as well as @jainkrati mentioned that he has more bandwidth than you @mohammadranjbarz, maybe you can delegate to him

**Comment by MohammadPCh** [comment]
@laurenluz @mohammadranjbarz,

We encountered another issue related to this, and I've explained it comprehensively there. We have the following images:

1. Background image
2. Sponsor image
3. Top right image
4. Top left image
5. Bottom right image
6. Bottom left image

Additionally, we need a standard for these images. I'm mentioning @rodricastillo9013 here because we can't move or resize images for each round; they should adhere to a standard.

**Comment by jainkrati** [comment]
@CarlosQ96 and @mateodaza pls work together to implement this before 10th Feb. We have Metapool round starting 19th Feb. @laurenluz pls correct if my understanding of dates is wrong.

**Comment by laurenluz** [comment]
you're pretty much right! only if we have this BEFORE feb 12, it is ok... Feb 12 is the last day for projects to apply, so will be the first day we start putting togehter the round on the UI... all admin bro changes should ideally be finished before Feb 12.

**Comment by laurenluz** [comment]
is this done? where are we with it? we need to update the metapool banner by Friday, so it I can't do it in admin bro, I need to create another issue

**Comment by CarlosQ96** [comment]
@laurenluz we haven't done backend work on this. It's not as simple, as we have to standardize the images. And it's not 1 image but multiple. 
Right now, Cherik or someone else has done the work on the frontend directly.

**Comment by laurenluz** [comment]
I am aware of how the banner usually gets updated. It also requires me having to do a lot of pushing & coordination, checking over and over on the issue to make sure we get the banner up "in time". I have to coordinate with @jainkrati and/or multiple devs back & forth every time we need to setup a new round, and my ability to do the rest of the set up in admin bro is always blocked by this frontend issue.

I'm not sure that it is 20x faster tbh if you consider the PM work involved.

**Comment by laurenluz** [comment]
We also are trying to release this feature where we can show archived QF rounds (with their corresponding banners)... #3748 and Cherik was needing somewhere to grab the images corresponding to QF rounds from, which I think is backend work that then relates also to this issue & makes it possible.

**Comment by jainkrati** [comment]
@RamRamez pls take both FE and BE

**Comment by jainkrati** [comment]
@maryjaf pls test

**Comment by maryjaf** [comment]
I've set a picture for banner of this active round https://impact-graph.serve.giveth.io/admin/resources/QfRound/records/90/edit
But it hasn't been changed, https://staging.giveth.io/qf/all
please take a look @Meriem-BM

**Comment by Meriem-BM** [comment]
Okay @maryjaf, the dynamic banner image is only implemented for archived rounds right now, will do that for active ones as well.

**Comment by maryjaf** [comment]
Thanks @Meriem-BM Yeah it works for archived round correctly
So let me know whenever for active round also this problem be resolved

**Comment by Meriem-BM** [comment]
@maryjaf you can check this now

**Comment by maryjaf** [comment]
Thanks @Meriem-BM It works for both archived and current qf rounds
